Sources/CouchbaseLiteSwift/CouchbaseLiteSwift.swift

```diff
@@ -0,0 +1,3 @@
+struct CouchbaseLiteSwift {
```

Comment:
Couchbase Lite Swift package is a binary swift package. Can you explain why we need this file? If we really need it, can we make it an empty struct without the "Hello, World!"?